lua/kubectl/state.lua

```diff
@@ -121,18 +121,13 @@ function M.restore_session()
   end
 
   -- Restore state
-  M.ns = M.session.namespace
+  local context_table = M.session.contexts[M.getContext().contexts[1].name] or M.session.contexts.default
```

Comment:
I'd love some if checks on that get context, maybe in the getcontext it self where it returns a default value so we don't get any raceconditions when trying to retrieve that information..

Reply:
honestly would appreciate some help here 🙌🏽